types/custom-functions-runtime/index.d.ts

```diff
@@ -75,6 +75,17 @@ declare namespace CustomFunctions {
          * [Api set: CustomFunctionsRuntime 1.3]
          */
         parameterAddresses?: string[];
+
+        /**
+         * Indicates whether the function is invoked as part of the formula value preview.
+         * `isInValuePreview` is read-only and can't be set by a custom functions add-in.
+         * In Excel on Windows and Mac, this value is `true` if the function is invoked is to preview the formula value; otherwise it is `false`.
```

Comment:
The fact that that formula value preview is not implemented for Excel Online isn't really needed here. It still holds that it is true if invoked for formula value preview and false otherwise. I recommend removing line 82.